Fix - Is IP the correct abbreviation for Identity Provider #10883 (#11435)
dotnet/docs: docs/framework/security/claims-based-identity-model.md

  
 This diagram shows a Web site (the relying party application, RP) that has been configured to use WIF for authentication and a client, a web browser, that wants to use that site.  
  

> 1.  When an unauthenticated user requests a page their browser is redirected to the identity provider (IdP) pages.  

Reviewer: ```suggestion
1.  When an unauthenticated user requests a page, their browser is redirected to the identity provider (IdP) pages.  
```

> 2.  The IdP requires the user to present their credentials, e.g. username/password, Kerberos, etc.  

Reviewer: ```suggestion
2.  The IdP requires the user to present their credentials, such as username/password or Kerberos authentication.  
```
